seedfarmer/commands/_deployment_commands.py

```diff
@@ -47,6 +49,55 @@
 _logger: logging.Logger = logging.getLogger(__name__)
 
 
+def _clone_module_repo(git_path: str) -> str:
+    """Clone a git repo and return directory it is cloned into
+
+    Rather than reinventing the wheel, we implement the Generic Git Repository functionality introduced by
+    Terraform. Full documentation on the Git URL definition can be found at:
+    https://www.terraform.io/language/modules/sources#generic-git-repository
+
+    Parameters
+    ----------
+    git_path : str
+        The Git URL specified in the Module Manifest. Full example:
+        https://example.com/network.git//modules/vpc?ref=v1.2.0&depth=1
+
+    Returns
+    -------
+    str
+        The local directory within the codeseeder.out/ where the repository was cloned
+    """
+    git_path = git_path.replace("git::", "")
+    ref: Optional[str] = None
+    depth: Optional[int] = None
+    module_directory = ""
+
+    if "?" in git_path:
+        git_path, query = git_path.split("?")
+        query_params = parse_qs(query)
+        ref = query_params.get("ref", [None])[0]  # type: ignore
+        if "depth" in query_params and query_params["depth"][0].isnumeric():
+            depth = int(query_params["depth"][0])
+
+    if ".git//" in git_path:
+        git_path, module_directory = git_path.split(".git//")
+
+    repo_directory = git_path.replace("https://", "").replace("git@", "").replace("/", "_").replace(":", "_")
+
+    working_dir = os.path.join(
+        os.getcwd(), "seedfarmer.gitmodules", f"{repo_directory}_{ref.replace('/', '_')}" if ref else repo_directory
```

Comment:
Shouldn't this be config.OPS_ROOT?  `seedfarmer.yaml` location has been established.

Reply:
yep. this should be `config.OPS_ROOT`. I mimicked the way `codeseeder` was creating it's codeseeder.out/ directory, but that can be a little inconsistent for a module cache.